### ISSUE TITLE
Installation via docker-compose fails due to python version conflicts (3.6 vs 3.8 required)

### ISSUE THREAD

**Issue by sally14** [doccano/doccano#1106]
Hello, 
First, thank you for developing and maintaining Doccano, which is a super-useful ressource. 

When trying to install the latest version of doccano via the docker-compose installation, I ran in a bug which forces the backend to exit, due to a python version conflict. 

I first did a canonical docker + docker compose installation (following docker documentation), then, following Doccano documentation :  

```bash
git clone https://github.com/doccano/doccano.git
cd doccano
docker-compose -f docker-compose.prod.yml up
```
I ran into this error: 

```bash
Starting doccano_postgres_1 ... done
Starting doccano_frontend_1 ... done
Starting doccano_backend_1  ... done
Starting doccano_nginx_1    ... done
Attaching to doccano_postgres_1, doccano_frontend_1, doccano_backend_1, doccano_nginx_1
backend_1   | Making staticfiles
backend_1   | Executing collectstatic
backend_1   | Traceback (most recent call last):
backend_1   |   File "manage.py", line 15, in <module>
backend_1   |     execute_from_command_line(sys.argv)
backend_1   |   File "/usr/lib/python3.6/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
backend_1   |     utility.execute()
backend_1   |   File "/usr/lib/python3.6/site-packages/django/core/management/__init__.py", line 325, in execute
backend_1   |     settings.INSTALLED_APPS
backend_1   |   File "/usr/lib/python3.6/site-packages/django/conf/__init__.py", line 79, in __getattr__
backend_1   |     self._setup(name)
backend_1   |   File "/usr/lib/python3.6/site-packages/django/conf/__init__.py", line 66, in _setup
backend_1   |     self._wrapped = Settings(settings_module)
backend_1   |   File "/usr/lib/python3.6/site-packages/django/conf/__init__.py", line 157, in __init__
backend_1   |     mod = importlib.import_module(self.SETTINGS_MODULE)
backend_1   |   File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
backend_1   |     return _bootstrap._gcd_import(name[level:], package, level)
backend_1   |   File "<frozen importlib._bootstrap>", line 994, in _gcd_import
backend_1   |   File "<frozen importlib._bootstrap>", line 971, in _find_and_load
backend_1   |   File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
backend_1   |   File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
backend_1   |   File "<frozen importlib._bootstrap_external>", line 674, in exec_module
backend_1   |   File "<frozen importlib._bootstrap_external>", line 781, in get_code
backend_1   |   File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
backend_1   |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
backend_1   |   File "/app/app/settings.py", line 283
backend_1   |     if (spec := importlib.util.find_spec(name)) is not None:
backend_1   |              ^
backend_1   | SyntaxError: invalid syntax
doccano_backend_1 exited with code 1
doccano_frontend_1 exited with code 0
```
It seems like the python version installed by docker was python 3.6, where, when I checked the main Dockerfile ([doccano/Dockerfile](https://github.com/doccano/doccano/blob/master/Dockerfile)), it should be python 3.8.6 (and manage.py syntax error suggests it is python 3.8). 

Might it be the alpine 3.9.6 version of [doccano/app/Dockerfile](https://github.com/doccano/doccano/blob/master/app/Dockerfile) which has python 3.6.9 by default? ( https://pkgs.alpinelinux.org/package/v3.9/main/x86_64/python3 )

Thank you in advance!

 Environment
---------

*   Operating System: Ubuntu 18.04.5 LTS
*   Python Version Used: 3.6.9 (but I don't use the python installation)
*   When you install doccano: December 10th, 2020
*   How did you install doccano (Heroku button etc): Docker-compose installation

**Comment by Hironsan** [comment]
Fixed in #1008 . Please rebuild images and run containers.

**Comment by sally14** [comment]
Thank you again!
